Sources/CryptorECC/ECPrivateKey.swift

```diff
@@ -59,8 +60,78 @@ public class ECPrivateKey {
     #endif
     let nativeKey: NativeKey
     let pubKeyBytes: Data
+    private var stripped: Bool = false
 
 
+    /**
+     Initialise an new ECPrivate key from a supported `Curve`
+     ### Usage Example:
+     ```swift
+     let key = try ECPrivateKey(for: .prime256v1)
+     ```
+     - Parameter for curve: The elliptic curve that is used to generate the key.
+     - Returns: An ECPrivateKey.
+     - Throws: An ECError if the key fails to be created.
+    */
+    public init(for curve: EllipticCurve) throws {
```

Comment:
As per previous comment, make this private and then supply a factory method that returns a new instance.